### PR TITLE
Added jQuery Ajax Native to Javiscript Libraries and Frameworks

### PR DESCRIPTION
**Pull request by tarikbenmerar**
@youknowriad  please review. Thanks in advance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Arli](https://github.com/elkebirmed/arli) - JavaScript tools for an Arabic development
   - [react-graphql-redux](https://github.com/youknowriad/react-graphql-redux) - use GraphQL to query your Redux store
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
-
+  - [jQuery Ajax Native](https://github.com/acigna/jquery-ajax-native) - jQuery Plugin that adds XHR2 responseType support
```
Comment (youknowriad) [inline README.md]: Thanks for this addition @tarikbenmerar if you could just keep the alphabetical order here and move this right after "Arli". Tks a lot

**Review by youknowriad — CHANGES_REQUESTED** [review]

**Comment by tarikbenmerar** [comment]
@youknowriad can you check now please ?

**Review by youknowriad — APPROVED** [review]

**Comment by youknowriad** [comment]
Cool! thanks for this addition @tarikbenmerar

**Comment by tarikbenmerar** [comment]
@youknowriad thank you for having such a central repo. Hope others will put their stuff here. Thanks again.

**Comment by youknowriad** [comment]
@tarikbenmerar you're welcome, I hope this will encourage collaboration as well :)

**Comment by tarikbenmerar** [comment]
@youknowriad hope so 👍

**Pull request merged**